docs/go-run-your-first-app.md

```diff
@@ -43,29 +47,29 @@ To run the application you will do the following:
 
 Here's a high-level illustration of what's happening:
 
-![High level project design](/img/docs/temporal-high-level-application-design.png)
+![High level project design](https://raw.githubusercontent.com/temporalio/documentation-images/main/static/temporal-high-level-application-design.png)
 
 ### The Workflow function
 
-When you "start" a Workflow you are basically telling the Temporal server, "track the state of the Workflow with this signature". Workers will execute the Workflow code below, piece by piece, relaying the execution events and results back to the server.
+The Workflow function is the application entry point. Below you can see how the money transfer Workflow looks.
```

Comment:
"Below you can see how the money transfer Workflow looks."

Bit awkward phrasing